src/analytics.ts

```diff
@@ -0,0 +1,18 @@
+declare let gtag: Function;
+
+export const EVENT_ACTION = "action";
+export const EVENT_EXIT = "exit";
+export const EVENT_CHANGE = "change";
+export const EVENT_SHAPE = "shape";
+
+export const trackEvent = gtag
```

Comment:
Any idea how to properly use the `gtag` here, that is defined in: https://github.com/excalidraw/excalidraw/blob/events/public/index.html#L87-L99